blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,112 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+
+runNw('./node_modules/nw/bin/nw');
+
+function runNw(nwPath){
+    var nw;
+    if(os.platform() === 'linux'){
+        // TODO - What happens to folks running a linux desktop?
+        nw = exec('./travis_runner.sh ' + nwPath + ' ./dist/tests');
```

Comment:
Agreed. This is actually not necessary. Something remaining when I was having trouble with travis and debugging with a shell script. This is removed